addons/mail/models/res_partner.py

```diff
@@ -236,7 +236,7 @@ def _field_store_repr(self, field_name):
         return [field_name]
 
     def _to_store_defaults(self):
-        return ["active", "avatar_128", "email", "im_status", "is_company", "name", "user"]
+        return ["active", "avatar_128", "email", "im_status", "is_company", "in_call", "name", "user"]
```

Comment:
Do we really need this as a default field?
It feels only useful in the invite livechat context.
I'd go as far as saying whether someone is in call is a private information that shouldn't be leaked unless absolutely necessary.